pennylane/ops/functions/is_commuting.py

```diff
@@ -374,165 +324,55 @@ def is_commuting(operation1, operation2):
     if operation2.name == "ControlledOperation" and operation2.control_base == "MultipleTargets":
         raise qml.QuantumFunctionError(f"{operation2.control_base} controlled is not supported.")
 
+    # Case 1 operations are disjoints
+    if not intersection(operation1.wires, operation2.wires):
```

Comment:
Moved up because we want to perform this simple, easy check before doing unnecessary simplification.